user/tests.py

```diff
@@ -1,3 +1,36 @@
-from django.test import TestCase
+import json
 
-# Create your tests here.
+from django.test    import TestCase, Client
+
+from user.models    import User
+
+class UnitTests(TestCase):
```

Comment:
```suggestion
class UserSignInTest(TestCase):
```
너무 포괄적인 이름이니까 이렇게 바꿨으면 좋겠습니다.

Reply:
class 이름을 구체화할 생각을 하지 못했었습니다. 감사합니다!